omnibus/files/private-chef-ctl-commands/chef-server-ctl

```diff
@@ -216,6 +216,17 @@ EOM
         "/admin/ping?wt=json"
       end
     end
+
+    # Override reconfigure to skip license checking
```

Comment:
The current implementation in omnibus-ctl that does the license checking has no special cases for open source projects.  If _any_ license is present it requires acceptance.  I suggested several possible solutions to the omnibus maintainers and they suggested this solution. I'm more than happy to implement something else but this is what was suggested.